styles.js

```diff
@@ -23,6 +23,7 @@ export default createGlobalStyle`
   body {
     margin: 0;
     font-family: system-ui;
+    font-size: 16px;
```

Comment:
Your font-size should not be in px! [See MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/font-size#pixels)